src/get-started/install/_get-sdk-win_v2.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Windows install
+description: How to install on Windows v2.
+short-title: Windows
+next:
+  title: Set up an editor
+  path: /get-started/windows-quick
+---
```

Comment:
**issue:** This text is displayed in the [staged version](https://flutter-website-jr-stagi-988d4.web.app/get-started/install/windows_quick).

Reply:
Still an issue.

![image](https://user-images.githubusercontent.com/706219/199070732-9ac6009b-6ad7-4a67-86b4-16980f57490c.png)